txpipe/maps.py

```diff
@@ -436,7 +439,9 @@ def run(self):
         mask[mask == healpy.UNSEEN] = 0
         mask[np.isnan(mask)] = 0
         mask = mask.flatten()
-        pix = np.where(mask > 0)[0]
+        #identify pixels to keep (i.e. mask value above specified threshold)
+        pix_keep = mask > self.config['weight_thresh']
+        pix = np.where(pix_keep)[0]
```

Comment:
do we use `pix` for anything?

Reply:
It gets included in the output file but isn't used for any of the calculations in TXDensityMaps.